src/Peachpie.Library/OpenSSL.cs

```diff
@@ -0,0 +1,741 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Pchp.Core;
+using System.Security.Cryptography;
+using System.IO;
+using static Pchp.Library.PhpHash;
+using System.Security.Cryptography.X509Certificates;
+using Pchp.Core.Utilities;
+
+namespace Pchp.Library
+{
+    /// <summary>
+    /// PHP openssl support.
+    /// </summary>
+    [PhpExtension("openssl")]
+    public static class OpenSSL
+    {
+        #region Constants
+        public const int OPENSSL_RAW_DATA = (int)Options.OPENSSL_RAW_DATA;
+        public const int OPENSSL_ZERO_PADDING = (int)Options.OPENSSL_ZERO_PADDING;
+
+        private static Dictionary<string, Cipher> Ciphers = new Dictionary<string, Cipher>
```

Comment:
is it case sensitive?

Reply:
Yes, it is. I have already corrected it.